src/components/Icon/Icon.tsx

```diff
@@ -1,30 +1,33 @@
 import React from 'react';
 
-// import iconTypes from './iconTypes';
-import CircleCross from './components/CircleCross';
+import iconMapping from './iconMapping';
 
-type IconProps = {
+type IconType = {
```

Comment:
Any reason this is called `IconType` as opposed to `IconProps`? Is it confusing that `ButtonProps` are still called `ButtonProps` (consistency)?

Reply:
True we should be consistent. There's no consensus of opinion so I suggest the following

If the type definition is describing component props we use `...Props` suffix and otherwise use `...Type` to describe anything else

Reply:
Sounds good to me - what would the "anything else" be, out of curiosity? Parameters for a function?

Reply:
Could be, or any variable we need to declare which TS flags up as an issue